kebechet/overlays/test/kebechet.yaml

```diff
@@ -1,122 +0,0 @@
----
```

Comment:
please also remove this 
https://github.com/thoth-station/thoth-application/blob/9b7e78feeeb19a90d62deeb69ebb6adde1499fd1/kebechet/overlays/test/kustomization.yaml#L11